timesketch/migrations/versions/b11dea30835c_.py

```diff
@@ -0,0 +1,46 @@
+"""empty message
+
+Revision ID: b11dea30835c
+Revises: 64c00337b9d1
+Create Date: 2020-04-02 14:40:42.286566
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = 'b11dea30835c'
+down_revision = '64c00337b9d1'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        'aggregation',
+        sa.Column('aggregationgroup_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(
+        None, 'aggregation', 'aggregationgroup',
+        ['aggregationgroup_id'], ['id'])
+    op.drop_column('aggregationgroup', 'agg_type')
```

Comment:
Hmm, I wonder if these are needed, or if it is because of previous interation of your DB migration?

Reply:
or wait.. did we have the model before?

Reply:
no, this is needed, since I'm adding aggregationgroup into the database for other modules...

is this drop column needed? I don't know, this is automatically generated code, I just assume this is done "just in case"

Reply:
file dopped... new one replaces it